docs/architecture/managers/epoch-manager.md

```diff
@@ -0,0 +1,131 @@
+# Epoch Manager
```

Comment:
I would perhaps update the [configuration toml documentation](https://github.com/witnet/witnet-rust/blob/master/docs/configuration/toml-file.md) and the [configuration env documentation](https://github.com/witnet/witnet-rust/blob/master/docs/configuration/environment.md) with the new fields.

Reply:
Done!